security/basic/pom.xml

```diff
@@ -13,12 +13,16 @@
     <dependencies>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-resteasy</artifactId>
+            <artifactId>quarkus-resteasy-reactive</artifactId>
```

Comment:
it should be `quarkus-rest` (no reactive in name); I think Fedor just want to avoid changes in backports :-) I would prefer to avoid `quarkus-resteasy-reactive` in main branch, no other module uses it. But I am not a reviewer here, so I'll leave you to decide.

Reply:
Still IMHO the issue should be linked here, or put some comment here, why it needs to be reactive. Otherwise someone will be very confused some day, when changing something here.